_episodes/03-configuration.md

```diff
@@ -20,20 +20,33 @@ keypoints:
 
 ## The configuration file
 
+For the purposes of this tutorial, we will create a directory in our home directory
+called `esmvaltool_tutorial` and use that as our working directory. The following steps 
+should do that:
+
+~~~bash
+ mkdir esmvaltool_tutorial
+ cd esmvaltool_tutorial
+~~~
+
 The ``config-user.yml`` configuration file contains all the global level
 information needed by ESMValTool to run.
 This is a [YAML file](https://yaml.org/spec/1.2/spec.html).
 
 You can get the default configuration file by running:
 
 ~~~bash
-  esmvaltool config get_config_user
+  esmvaltool config get_config_user --path=<target_dir>
 ~~~
-
-It will save the file to: `~/.esmvaltool/config-user.yml`, where `~` is the
+The default configuration file will be downloaded to the directory specified with 
+the `--path` variable. For instance, you can provide the path to your working directory 
+as the `target_dir`. If this option is not used, the file will be saved to the default 
+location: `~/.esmvaltool/config-user.yml`, where `~` is the
 path to your home directory. Note that files and directories starting with a
```

Comment:
if a `config-user.yml` exists in `~/.esmvaltool/` and a user runs `esmvaltool config get_config_user`, the config file won't be updated because esmvaltool does not overwrite the config file. So, the user should first remove that config file manually. Shall we explain this here?

Reply:
OK. This is now done.